management/job.go

```diff
@@ -0,0 +1,25 @@
+package management
+
+import "time"
+
+type Job struct {
+	ID        *string    `json:"id,omitempty"`
+	Status    *string    `json:"status,omitempty"`
+	Type      *string    `json:"type,omitempty"`
+	CreatedAt *time.Time `json:"created_at,omitempty"`
+
+	UserID   *string `json:"user_id,omitempty"`
+	ClientID *string `json:"cliend_id,omitempty"`
```

Comment:
Can you add comments from the schema description in the management api docs?

For example 

```Go
// The job's identifier. Useful to retrieve its status
ID *string `json:"id,omitempty"`
```